### PR TITLE
Add missing make child active functionality

### PR DESCRIPTION
**Pull request by hamza-vd**
See [issue](https://github.com/onaio/opensrp-client-wellnesspass/issues/544)

### CODE REVIEW

File: opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java [hunk 1]
```diff
@@ -237,4 +239,55 @@ private CommonPersonObjectClient getChildDetails() {
 
         return commonPersonObjectClient;
     }
+
+    @Test
+    public void testGenderButtonColor() {
```
Comment (qiarie) [inline opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java:244]: Make the test case names in the format `test[MethodName][usecase/scenario]`

Comment (hamza-vd) [reply opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java]: Resolved

File: opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java [hunk 2]
```diff
@@ -738,6 +738,8 @@ private void showActivateChildStatusDialogBox() {
         final ActivateChildStatusDialogFragment activateChildStatusFragmentDialog = ActivateChildStatusDialogFragment.newInstance(thirdPersonPronoun, childCurrentStatus, R.style.PathAlertDialog);
         activateChildStatusFragmentDialog.setOnClickListener((dialog, which) -> {
             if (which == DialogInterface.BUTTON_POSITIVE) {
+                ChildDbUtils.updateChildDetailsValue(Constants.CHILD_STATUS.INACTIVE, String.valueOf(false), childDetails.entityId());
```
Comment (qiarie) [inline opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java:741]: Use the FALSE String constant already defined

File: opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java [hunk 3]
```diff
@@ -761,7 +763,7 @@ private void addServiceUndoDialogFragment(ServiceGroup serviceGroup, ServiceWrap
         undoServiceDialogFragment.setOnDismissListener(dialog -> dialogOpen = false);
     }
 
-    private String getChildsThirdPersonPronoun(CommonPersonObjectClient childDetails) {
+    protected String getChildsThirdPersonPronoun(CommonPersonObjectClient childDetails) {
```
Comment (qiarie) [inline opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java:766]: Why change the access modifier for the method?

File: opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java [hunk 4]
```diff
@@ -237,4 +239,68 @@ private CommonPersonObjectClient getChildDetails() {
...
+        Mockito.doReturn(0).when(fab).getPaddingRight();
+        Mockito.doReturn(0).when(fab).getPaddingTop();
+        baseChildImmunizationActivity.floatingActionButton = fab;
+        baseChildImmunizationActivity.configureFloatingActionBackground(0, null);
+
+        Mockito.verify(fab).setVisibility(View.VISIBLE);
+    }
+
+    @Test
+    public void testGetChildsThirdPersonPronounReturnsHimForMaleGender() {
```
Comment (qiarie) [inline opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java:287]: I don't think we needed to change the access modifier to test the method out

Comment (hamza-vd) [reply opensrp-child/src/test/java/org/smartregister/child/activity/BaseChildImmunizationActivityTest.java:288]: Resolved

File: opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java (hunk 3 above)
Comment (hamza-vd) [reply opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java]: Reverted

File: opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java (hunk 2 above)
Comment (hamza-vd) [reply opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildImmunizationActivity.java]: Done

**Review by qiarie — APPROVED** [review]

**Pull request merged**